src/Tribe/Tickets.php

```diff
@@ -4217,6 +4217,24 @@ public function __toString() {
 			return $this->class_name;
 		}
 
+		/**
+		 * Return the URL slug to be used by the provider method.
+		 *
+		 * @since TBD
+		 *
+		 * @return string
+		 */
+		public static function get_provider_url_slug(){
```

Comment:
Asking the question, because I don't know, but is there a better place for this method? This file is already 4000+ lines long, there might be a better provider class to put this in?

```suggestion
		public static function get_provider_url_slug() {
```